### ISSUE TITLE
Cannot use Dataform tools in WSL Ubuntu

### ISSUE THREAD

**Issue by msnillo** [ashish10alex/vscode-dataform-tools#9]
Hi @ashish10alex,

I followed all of the instructions in `Requirements` section but still was not able to use the extension. 

I got this error:
`command 'vscode-dataform-tools.formatCurrentfile' not found`

And here's the screenshot of runtime status of the extension:
![image](https://github.com/user-attachments/assets/94927047-d201-4966-b194-fdbff388b1ce)

Are there any steps I'm missing?
Thank you!

**Comment by ashish10alex** [comment]
Hi @msnillo , 
Thanks for raising the issue.  `vscode-dataform-tools.formatCurrentfile` relies on a cli  [formatdataform](https://github.com/ashish10alex/formatdataform) to be installed. Can you please confirm if this is installed.  If you have already installed this can you please send the output of the following 

1. `formatdataform --version`
2. Have you open the workspace in the root of your dataform project ?
3. Can you try to use the menu bar as shown in [this gif](https://github.com/ashish10alex/vscode-dataform-tools/blob/main/media/images/formatting.gif) ( if that is there ) and send me the error log which would come at the bottom right 
4. Is this the only command that you are not able to use in the extension ?

**Comment by msnillo** [comment]
Hi @ashish10alex, thanks for replying and sorry for the late revert.

To answer your previous questions:
1. Yes, I have installed `formatdataform`
2. No, it was not in the root of my dataform project. Currently we splitted our Dataform projects to multiple repo since we have a lot of scripts to maintain. This actually solved the usage of formatting sqlx files for me, plus the reloading of the WSL instance I have.
3. Resolved in (2)
4. Upon checking I cannot access other features like compiled query, autocompletion, or going to definition. I always get this error pop-up, although I'm already in the root directory of my dataform project:
![image](https://github.com/user-attachments/assets/17fcf9db-38f1-4179-8acb-bb11e90125e9)

**Comment by ashish10alex** [comment]
Thanks @msnillo , for your detailed feedback. The error in (4) is something I have not seen yet. 
Can you please try this repro repo and see if you are able to reproduce the error that you are getting in your personal repository ? 
Also to make sure, you are using the latest version of the extension right ? - `v0.0.35` ? 

Repro repo:  https://github.com/ashish10alex/test_dataform

I have added the steps you need to take as well to make sure we are able to identify the issue 

![CleanShot 2024-08-12 at 08 15 16@2x](https://github.com/user-attachments/assets/7073c9ca-20eb-48f6-b40c-d4097e1d85c0)

**Comment by msnillo** [comment]
I was able to resolve the access issue I encountered. I just needed to open the dataform folder in a separate window, instead of having a /home folder where multiple dataform repos are open, then reload the windows and wait for the extension to work.

I was able to resolve the issue with the help of your guide and additional documentation - it was a great help especially during the installlation and testing.
